src/main/kotlin/io/inventi/eventstore/eventhandler/IdempotentPersistentSubscriptionListener.kt

```diff
@@ -181,28 +189,27 @@ internal class IdempotentPersistentSubscriptionListener(
         }
     }
 
+    private fun RecordedEvent.toEventIdsOrMetadata(method: Method, parameterNumber: Int) =
+            if (method.parameters[parameterNumber].type.simpleName == EventIds::class.java.simpleName) {
+                EventIds(originalId = originalEventIdOrNull, effectiveId = effectiveEventId)
```

Comment:
So if i understand this correctly, if event contains `IdempotentEventHandler.OVERRIDE_EVENT_ID`, these values will be equal?
It should actually be:
```
val originalEventId: String? = OVERRIDE_EVENT_ID from metadata
val eventId: String =  RecordedEvent.eventId.toString()
val effectiveEventId get() = originalEventId ?: eventId
```

Reply:
Think should be called:
EventIds(overridden = overriddenEventIdOrNull, current = eventId.toString(), effective = effectiveEventId)